recipes/serpenttools/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "serpentTools" %}
+{% set version = "0.10.1" %}
+
+package:
+  # Do need the package name to be lower cased here, even though the
+  # package has that dang upper case T
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: da6fbd0db6a55538720a1150cbd146c94dd5049f75b8bd1f9eb83c30c213360b
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - matplotlib-base >=2.2.3,!=3.3.4
+    - numpy >=1.16.0
+    - pip
+    - python
+    - pyyaml >=5.1.1
```

Comment:
These seem to be incorrect: https://github.com/CORE-GATECH-GROUP/serpent-tools/blob/ec46c96cc98e19fb83c5c205ee4057f3d951064e/pyproject.toml#L1

Reply:
This was missing the build dependencies, which `grayskull` caught. The other runtime dependencies should be correct